File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.js
/**
 * @param {number} n
 * @return {number[]}
 */
var sumZero = function(n) {
    let output = [];
    
    for(let i = 1; i < Math.floor(n/2)+1; i++) {
        output.push(i);
        
        output.push(-i);
    }
    
    if(n % 2 != 0) {
        output.push(0)
    }
    return output
    
};

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.cs
public class Solution {
    public int[] SumZero(int n) {
        List<int> output = new List<int>();
        double loopSize = Math.Floor((double)n/2) +1;
        
        for(int i = 1; i < loopSize; i++) {
            output.Add(i);
            output.Add(-i);
        }
        if(n % 2 != 0) {
            output.Add(0);
        }
        return output.ToArray();
    }
}